frontend/packages/console-shared/src/components/status/Status.tsx

```diff
@@ -92,6 +93,9 @@ export const Status: React.FC<StatusProps> = ({
     case 'Unknown':
       return <StatusIconAndText {...statusProps} icon={<UnknownIcon />} />;
 
+    case 'PipelineNotStarted':
+      return <StatusIconAndText {...statusProps} icon={<OutlinedHourglassIcon />} />;
```

Comment:
Created a Story to update this icon now they have it figured out. https://issues.redhat.com/browse/ODC-5047